server/app/routes/role.py

```diff
@@ -0,0 +1,15 @@
+"""Endpoints for Role database model."""
+
+from flask import Blueprint
+
+from app.domain_logic import role_domain_logic
+from app.schemas.user import RoleSchema
+
+ROLE_BP = Blueprint('role_bp', __name__, url_prefix='/role')
+
+
+@ROLE_BP.route('/', methods=["GET"])
+def get_roles():
+    roles = role_domain_logic.get_roles()
+    serialized_roles = [RoleSchema().dump(role) for role in roles]
+    return {"roles": serialized_roles}
```

Comment:
Was going to say we should ensure the response MIME type is being set, but REST Client shows it coming back with an application/json. Not sure if Marshmallow does that for us if Flask is smart enough to figure it out.